rasa_core/actions/action.py

```diff
@@ -157,6 +159,18 @@ def __str__(self) -> Text:
         return "UtterAction('{}')".format(self.name())
 
 
+class ActionBack(Action):
+    """Revert the tracker state by two user utterances."""
+
+    def name(self) -> Text:
+        return ACTION_BACK_NAME
+
+    def run(self, dispatcher, tracker, domain):
+        from rasa_core.events import UserUtteranceReverted
```

Comment:
don't need to add this import statement, it already gets imported at the start

Reply:
i removed the import in the latest commit